### DISCUSSION TITLE
Numbers drop down under the nature

### DISCUSSION THREAD

**Discussion by KingMarzy** [kwsch/PKHeX#3650] [Q&A] [answered]
What do these do? I can’t seem to find an explanation or I just don’t know where to look.

![image](https://user-images.githubusercontent.com/63757067/204103300-b852f6a4-b562-419d-8224-01fb4eeede3b.jpg)

**Comment by sora10pls** [comment, marked as answer]
Form argument. Used to store values related to evolution criteria.

Examples include how many times Primeape has used Rage Fist, how many coins were used to evolve Gimmighoul, how much HP Galarian Yamask has lost in battle…

**Reply by KingMarzy** [reply]
Thanks, that helps alot!
